tests/decode-erspan-typeI-03/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
+
+  min-version: 5.0.0
+
+
+args:
+    - --set decoder.erspan.typeI.enabled=false
+
+checks:
+
+    - filter:
+        count: 0
+        match:
+            event_type: flow
+
+    - stats:
+        decoder.erspan: 0
```

Comment:
This test does not pass even if the suricata PR was merged.
`Sub test #2: FAIL : stats.decoder.erspan: expected 0; got 42`
What is wrong ? The test or suricata ?